front_end/src/components/layout/Footer.js

```diff
@@ -1,50 +1,30 @@
 import React from "react";
+import "./Footer.css";
 import { Link } from "react-router-dom";
+import { withStyles, Grid } from "@material-ui/core";
```

Comment:
please remove withStyles at import and export down below if you are not using it

Reply:
good catch, thanks